content/docs/command-reference/plots/show.md

```diff
@@ -1,58 +1,72 @@
-                     [-s SELECT] [-x X] [-y Y] [--stdout]
-                     [--no-csv-header] [--no-html] [--title TITLE]
-                     [--xlab XLAB] [--ylab YLAB] [datafile]
+usage: dvc plots show [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
+                      [-y <field>] [--no-header] [--title <text>]
+                      [--x-label <text>] [--y-label <text>] [-o <path>]
+                      [--show-vega]
+                      [targets [targets ...]]
 
 positional arguments:
-  datafile              Metrics file to visualize
+  targets               Metric files to visualize.
+                        Shows all plots by default.
 ```
 
 ## Description
 
 This command provides a quick way to visualize metrics such as loss functions,
-AUC curves, confusion matrices, etc. Please see `dvc plots` for information on
-the supported data formats and other relevant details about DVC plots.
+AUC curves, confusion matrices, etc. All plots defined in `dvc.yaml` are used by
+default.
+
+Optionally, specific metric file `targets` to show are accepted. These must be
+listed in a [`dvc.yaml`](/doc/user-guide/dvc-file-format) file (see the
+`--plots` option of `dvc run`).
+
+The plot style can be customized with
+[plot templates](/doc/command-reference/plots#plot-templates), using the
+`--template` option. To learn more about metric file formats and templates
+please see `dvc plots`.
+
+> Note that the default behavior of this command can be modified per metrics
+> file with `dvc plots modify`.
 
 ## Options
 
-- `-t [TEMPLATE], --template [TEMPLATE]` -
+- `-o <path>, --out <path>` - name of the generated file. By default, the output
+  file name is equal to the input filename with a `.html` file extension (or
+  `.json` when using `--show-vega`).
+
+- `-t <name_or_path>, --template <name_or_path>` -
   [plot template](/doc/command-reference/plots#plot-templates) to be injected
   with data. The default template is `.dvc/plots/default.json`. See more details
   in `dvc plots`.
 
-- `-f FILE, --file FILE` - name of the generated file. By default, the output
-  file name is equal to the input filename with additional `.html` suffix or
-  `.json` suffix for `--no-html` mode.
-
-- `--no-html` - do not wrap output Vega specification (JSON) with HTML.
-
-- `-x X` - field name for X axis. An auto-generated `index` field is used by
-  default.
+- `-x <field>` - field name from which the X axis data comes from. An
+  auto-generated `index` field is used by default. See
+  [Custom templates](/doc/command-reference/plots#custom-templates) for more
+  information on this `index` field. Column names or numbers are expected for
+  tabular metric files.
 
-- `-y Y` - field name for Y axis. The last column or field found in the
-  `datafile` is used by default.
+- `-y <field>` - field name from which the Y axis data comes from. The last
+  field found in the `targets` is used by default. Column names or numbers are
```

Comment:
indices?

Reply:
Idk. We already have the name `index` for an auto-generated number in the X axis. Should we overload the term for Y axis column numbers too?

Reply:
column name
column index

these ^^ are established terms

`number of column number`  - not correct or meaningful

> have the name index for an auto-generated number in the X axis. Should we overload the term for Y axis column numbers too?

we can be explicit - `column index` and even put an explanation (e.g. when `no-header` options is specified)

Reply:
Extracting to #1414